src/utils/Processing.ts

```diff
@@ -564,6 +576,44 @@ export function writeFHIRResources(
   writeResources(outPackage.instances.filter(i => i._instanceMeta.usage !== 'Inline'));
 
   logger.info(`Exported ${count} FHIR resources as JSON.`);
+  return { skippedResources };
+}
+
+export function writeFSHIndex(
+  outDir: string,
+  outPackage: Package,
+  inputDir: string,
+  skippedResources: string[] = []
+) {
+  const textIndex: string[][] = [];
+  const jsonIndex: FshFhirMapping[] = [];
+  [...outPackage.fshMap.keys()]
+    .filter(fileName => !skippedResources.includes(fileName))
+    .sort()
+    .forEach(fileName => {
+      const fshInfo = outPackage.fshMap.get(fileName);
+      const relativeInput = path.relative(inputDir, fshInfo.file);
+      textIndex.push([
+        fileName,
+        fshInfo.fshName,
+        fshInfo.fshType,
+        relativeInput,
+        `${fshInfo.location.startLine} - ${fshInfo.location.endLine}`
+      ]);
+      jsonIndex.push({
+        fshFile: relativeInput,
+        fshName: fshInfo.fshName,
+        fshType: fshInfo.fshType,
+        startLine: fshInfo.location.startLine,
+        endLine: fshInfo.location.endLine,
+        outputFile: fileName
```

Comment:
I can't decide if it's silly to ask for this or not, but... what do you think about ordering these properties in the same order as for the text file? (E.g., outputFile, fshName, fshType, fshFile, startLine, endLine).